monocle/db/queries.py

```diff
@@ -619,3 +610,26 @@ def oldest_open_changes(es, index, repository_fullname, params):
     data = run_query(es, index, body)
     changes = [r['_source'] for r in data['hits']['hits']]
     return {'items': changes, 'total': data['hits']['total']}
+
+
+def all_changes(es, index, repository_fullname, params):
```

Comment:
I would have preferred "changes_and_events" ?

Reply:
ok changing it

Reply:
done